mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -254,7 +254,13 @@ object AutoImports extends AutoImportsBackticks:
         imports: List[ImportSel]
     )(using Context): Option[l.TextEdit] =
       if imports.nonEmpty then
-        val indent = " " * importPosition.indent
+        val indent0 = " " * importPosition.indent
+        val editPos = pos.withSpan(Spans.Span(importPosition.offset)).toLsp
+        val indent =
+          if pos.source.path.isWorksheet &&
+            editPos.getStart().getCharacter() == 0
+          then indent0.drop(2)
```

Comment:
I'd recommend adding a comment here saying why we need to drop 2, since it ends up being wrapped.

Reply:
Actually, why don't we remove the indentation with AdjustData?  `worksheetScala3AdjustmentsForPC` should already do it :thinking:

Reply:
Currently `AdjustLspData` adjusts only the positions, not inserted text, so `worksheetScala3AdjustmentsForPC` for imports gives us `char = -2`, without deleting the indent. I'm not sure why it worked before, but here https://github.com/scalameta/metals/pull/4853 we align these adjustments to 0. I think allowing `AdjustLspData` to adjust insert text would be much bigger change

Reply:
Let's at least at a test to the ImportMissingSymbolLspSuite so that this is caught if we ever change the behaviour. Also add a note there, so people know where to look.

Reply:
Is `ImportMissingSymbolLspSuite` the right place? This issue is specific to worksheets, but also I haven’t seen any tests for completion edits

Reply:
But that happens on using auto imports in worksheets though? You would just need to change a name of the file under test